linux_os/guide/system/software/updating/security_patches_up_to_date/rule.yml

```diff
@@ -60,21 +60,19 @@ references:
     cis-csc: 18,20,4
     anssi: BP28(R08)
 
-{{# Make sure all the external OVAL content links are secured via TLS! #}}
 
 # SCAP 1.3 content should reference flat non compressed xml files
-{{% if product == "rhel7" %}}
-oval_external_content: "https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml"
-{{% elif product == "rhel8" %}}
-oval_external_content: "https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL8.xml"
-{{% elif product == "ubuntu1604" %}}
-oval_external_content: "https://people.canonical.com/~ubuntu-security/oval/com.ubuntu.xenial.cve.oval.xml"
-{{% elif product in ["ol7", "ol8"] %}}
-oval_external_content: "https://linux.oracle.com/security/oval/com.oracle.elsa-all.xml.bz2"
-{{% elif product == "sle12" %}}
-oval_external_content: "https://support.novell.com/security/oval/suse.linux.enterprise.12.xml"
-{{% elif product == "sle15" %}}
-oval_external_content: "https://support.novell.com/security/oval/suse.linux.enterprise.15.xml"
+{{% if oval_feed_url %}}
+oval_external_content: {{{ oval_feed_url }}}
+  {{% if not oval_feed_url.startswith("https") %}}
+  {{{ raise("OVAL feed of product '" + product + "' is not available through an encrypted channel: " + oval_feed_url) }}}
+  {{% endif %}}
+  {{% if not oval_feed_url.endswith(".xml") %}}
+warnings:
+    - general: "The OVAL feed of {{{ full_name }}} is not a XML file, which may not be understood by all scanners."
+  {{% endif %}}
 {{% else %}}
 {{# The rule will be "notchecked" #}}
+warnings:
+    - general: "{{{ full_name }}} doesn't have a OVAL CVE Feed associated, so this rule doesn't have a corresponding check."
```

Comment:
```suggestion
    - general: '{{{ full_name }}} doesn't have a corresponding OVAL CVE Feed. Therefore, this will result in a "not checked" result during a scan.'
```

Reply:
Thanks, I felt that something wasn't right with that sentence.